.gitignore

```diff
@@ -0,0 +1,43 @@
+# Whisk
```

Comment:
I think .gitignore is necessary. Because while I build Java sample with Gradle, some files will be generated and they don't need to submit to Git.